src/components/portfolio/grid.js

```diff
@@ -68,7 +68,7 @@ const PortfolioGrid = () => {
 			</div>
 		    <div className="grid__wrap">
 		      <div className="grid__container">
-		      	{ portfolioTiles.map( (tile, i) => ( <Tile props={ tile } key={i + 1} />) ) }
+		      	{ portfolioTiles.map( tile => <Tile {...tile} key={tile.slug} /> ) }
```

Comment:
Worth mentioning: It's not a good idea to use the index as `key`, as it's not unique and can cause issues in larger applications.